.releaserc.release.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
@BenjaminHalko 

I'm not sure how semantic release work *yet*, but can you please make the changelog format it matched with the one from v1.18.0

Reply:
Fixed in #1668